### PR TITLE
Add wif

### PR DESCRIPTION
**Pull request by RahulJain28**

### CODE REVIEW

File: master_config.json [hunk 1]
```diff
@@ -121,6 +121,12 @@
         "symbol": "EURC",
         "mint": "HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr",
         "logoUri": "https://www.circle.com/hubfs/Brand/EURC/EURC-icon_128x128.png"
+      },
+      {
+        "name": "dogwifhat",
+        "symbol": "WIF",
```
Comment (eugene-chen) [inline master_config.json:127]: it's $WIF

File: master_config.json [hunk 2]
```diff
@@ -121,6 +121,12 @@
...
+        "mint": "EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm",
+        "logoUri": "https://bafkreibk3covs5ltyqxa272uodhculbr6kea6betidfwy3ajsav2vjzyum.ipfs.nftstorage.link/"
```
Comment (eugene-chen) [inline master_config.json:129]: is there a more robust URL?

File: master_config.json (hunk 1 above)
Comment (eugene-chen) [reply master_config.json:127]: (annoying!)

File: master_config.json (hunk 2 above)
Comment (RahulJain28) [reply master_config.json:129]: that one is solscans
could also use coinmktcap
https://s2.coinmarketcap.com/static/img/coins/64x64/28752.png

Comment (eugene-chen) [reply master_config.json:129]: yeah this one is fine, cmc so small

**Review by eugene-chen — APPROVED** [review]

**Pull request merged**